notebook/notebook/notebook.py

```diff
@@ -131,10 +171,153 @@ def external_url_for(path):
     return url + path
 
 
+def forbidden():
+    return 'Forbidden', 403
+
+
 @app.route('/healthcheck')
 def healthcheck():
     return '', 200
 
+##### New notebook methods meant for consumption from agnostic client#####
+###### Methods that do not call the auth-gateway /verify method must not be directly public #######
+@app.route('/verify/<svc_name>/', methods=['GET'])
+def validate_get(svc_name):
```

Comment:
prefer the function name to mirror the route as closely as possible. probably `verify_get`

Reply:
Sure